db/dbformat.h

```diff
@@ -77,7 +79,7 @@ extern const ValueType kValueTypeForSeekForPrev;
 // (i.e. a type used in memtable skiplist and sst file datablock).
 inline bool IsValueType(ValueType t) {
   return t <= kTypeMerge || t == kTypeSingleDeletion || t == kTypeBlobIndex ||
-         kTypeDeletionWithTimestamp == t;
+         t == kTypeDeletionWithTimestamp || t == kTypeWideColumnEntity;
```

Comment:
I feel `kTypeDeletionWithTimestamp == t` is safer because you cannot mistaken it with `t = kTypeDeletionWithTimestamp`.

Reply:
Yeah, I just wasn't a fan of the inconsistency across value types :) Will change these to be the other way around.